### PR TITLE
Remove cli_common usage from pulselistener

### PR DESCRIPTION
**Pull request by La0**
First step toward lib split

Oversedes #2193

### CODE REVIEW

**Comment by La0** [comment]
@marco-c It should be easier to reviews this one commit-by-commit

File: src/pulselistener/pulselistener/monitoring.py [hunk 1]
```diff
@@ -222,3 +222,7 @@ def send_report(self):
 
         # Reset stats
         self.stats = {}
+
+
+# Shared monitoring manager
+task = Monitoring(7 * 3600)
```
Comment (marco-c) [inline src/pulselistener/pulselistener/monitoring.py]: I'd keep calling this `task_monitoring` so you don't have to do `import as`. I also like this name more than `task`

File: src/pulselistener/pulselistener/hook.py [hunk 2]
```diff
@@ -3,8 +3,8 @@
 
 import structlog
 
-from cli_common.pulse import create_consumer
```
Comment (marco-c) [inline src/pulselistener/pulselistener/hook.py]: Do we have other users of cli_commont.pulse? We could remove it if it's no longer used by anyone else (it was probably always only used by pulselistener)
Same with other libraries in cli_common, maybe pulselistener is now the only remaining user.

Comment (La0) [reply src/pulselistener/pulselistener/hook.py]: Unfortunately, no, shipit_api & tooltool_api are using these functions.

**Pull request merged**